src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -289,9 +289,9 @@ def csv(self, path: str) -> DataFrame:
         if self._session.sql_simplifier_enabled:
             df = DataFrame(
                 self._session,
-                SelectStatement(
-                    from_=SelectSnowflakePlan(
-                        self._session._plan_builder.read_file(
+                self._session._analyzer.create_SelectStatement(
```

Comment:
`create_SelectStatement` feels like a mixed usage of naming patterns. Should we do `create_select_statement` instead?

Reply:
updated